src/pages/trade/HorizontalPanel.tsx

```
@@ -51,7 +51,7 @@ import { MarketTypeFilter, PanelView } from './types';
 enum InfoSection {
   Position = 'Position',
   Orders = 'Orders',
-  OrderHistory = 'OrderHistory',
+    OrderHistory = 'OrderHistory',
```

ESLint

[error] Delete `··`

(prettier/prettier)